src/coverlet.core/Helpers/InstrumentationHelper.cs

```diff
@@ -153,8 +155,8 @@ public bool PortablePdbHasLocalSource(string module, out string firstNotFoundDoc
                         }
                         catch (BadImageFormatException)
                         {
-                            // TODO log this to warning
-                            // In case of non portable pdb we get exception so we skip file sources check
+                            _logger.LogWarning($"{nameof(BadImageFormatException)} during MetadataReaderProvider.FromPortablePdbStream in " +
```

Comment:
I don't like `+` can you update sentence all in one line?

Reply:
Done